sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_arg_dim.cpp

```diff
@@ -14,7 +14,7 @@
 
 // Waiting for the commit in IGC to be pulled into the driver to resolve the
 // test.
-// XFAIL: gpu
+// XFAIL: !igc-dev || gpu-intel-dg2
```

Comment:
@YixingZhang007 , XFAIL-TRACKER jira pointed below is submitted against PVC issue, that you fixed. If test is passing on PVC and is failing on DG2, it should be different jira, since problem is different. Could you please update xfail tracker?

Reply:
For sure! I will create a new Jira issue regarding this :)